projects/valtimo/plugin/src/lib/plugins/zaken-api/components/create-medewerker-zaak-rol/create-medewerker-zaak-rol.component.html

```diff
@@ -0,0 +1,120 @@
+<!--
+  ~ Copyright 2015-2025 Ritense BV, the Netherlands.
+  ~
+  ~ Licensed under EUPL, Version 1.2 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" basis,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<v-form
+  (valueChange)="formValueChange($event)"
+  *ngIf="{
+    disabled: disabled$ | async,
+    prefill: prefillConfiguration$ ? (prefillConfiguration$ | async) : null,
+  } as obs"
+>
+  <v-input
+    name="roltypeUrl"
+    [title]="'roltypeUrl' | pluginTranslate: pluginId | async"
+    [margin]="true"
+    [defaultValue]="obs.prefill?.roltypeUrl"
+    [disabled]="obs.disabled"
+    [required]="true"
+    [trim]="true"
+    [tooltip]="'roltypeUrlTooltip' | pluginTranslate: pluginId | async"
+    [fullWidth]="true"
+  >
+  </v-input>
+  <v-input
```

Comment:
There's some things in this PR that don't conform to the coding guidelines, like here a whitespace is inserted between children elements on the same level. Also, you are music public / private / readonly etc for a lot of properties in your Typescript classes. Can you go over your code again with this is mind? https://github.com/valtimo-platform/valtimo-frontend-libraries/blob/next-minor/CODING-GUIDELINES.md